src/main/java/picard/nio/PicardBucketUtils.java

```diff
@@ -66,13 +64,12 @@ public static String getTempFilePath(String prefix, String extension){
      * @param prefix The beginning of the file name
      * @param suffix The end of the file name, e.g. ".tmp"
      */
-    private static String randomRemotePath(String stagingLocation, String prefix, String suffix) {
+    private static Path randomRemotePath(String stagingLocation, String prefix, String suffix) {
         if (isGcsUrl(stagingLocation)) {
             // Go through URI because Path.toString isn't guaranteed to include the "gs://" prefix.
```

Comment:
I know this comment was already here, but I'm not sure what it refers to. I think its obsolete or something since it seems out or place (and possibly even wrong). I'd remove it.

Reply:
deleted